cmd/oras/internal/option/packer.go

```diff
@@ -69,6 +72,29 @@ func (opts *Packer) ExportManifest(ctx context.Context, fetcher content.Fetcher,
 	}
 	return os.WriteFile(opts.ManifestExportPath, manifestBytes, 0666)
 }
+func (opts *Packer) Parse() error {
+	var failedPaths []string
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+	if !opts.PathValidationDisabled && len(opts.FileRefs) != 0 {
+		for _, path := range opts.FileRefs {
+			//Remove the type if specified in the path <file>[:<type>] format
+			path, _, err = fileref.Parse(path, "")
+			if err != nil {
+				return err
+			}
+			if filepath.IsAbs(path) {
+				failedPaths = append(failedPaths, path)
+			}
+		}
+		if len(failedPaths) > 0 {
+			return fmt.Errorf("%v: %v currentDir :%v", errPathValidation, strings.Join(failedPaths, ", "), currentDir)
```

Comment:
Since we loosen the check to disallow absolute path, there is no need to output current path
```suggestion
			return fmt.Errorf("%w: %v", errPathValidation, strings.Join(failedPaths, ", "))
```